app/javascript/components/widgets/guests/panel.jsx

```diff
@@ -158,6 +162,7 @@ const Guests = ({ cityName }) => (
 );
 
 Guests.propTypes = {
+  startTimer: PropTypes.func.isRequired,
```

Comment:
This probably belongs in the previous commit enabling the announcement widget. Likely above too.